src/parachain/issue.ts

```diff
@@ -207,17 +203,30 @@ export class DefaultIssueAPI extends DefaultTransactionAPI implements IssueAPI {
 
     async request(
         amount: MonetaryAmount<WrappedCurrency, BitcoinUnit>,
-        vaultId?: AccountId,
+        vaultAccountId?: AccountId,
```

Comment:
I think those two arguments should be combined into one argument that is an object. Because either you submit both or none, but not just one of those.

Reply:
Given this reasoning, we might as well require the full `InterbtcPrimitivesVaultId` ( the tuple `[vaultAccountId, collateral, wrapped]`) instead of setting the wrapped currency in the API. If the UI is to know which vault has issuable tokens, it probably knows the full ID too.

Reply:
Let's do whatever is faster to implement, bear in mind that for the time being setting the wrapped currency ID will have no effect - we only support one per chain.